lib/CBDShapeExtractor.ts

```diff
@@ -4,6 +4,9 @@ import { Path, PathResult } from "./Path";
 import { DataFactory } from "rdf-data-factory";
 import { RdfStore } from "rdf-stores";
 import { Quad, Term } from "@rdfjs/types";
+import debug from "debug";
+
+const log = debug("cbdExtracted");
```

Comment:
I’d try to be more helpful here: cbdExtracted is ambiguous. Something like: `Member being extracted using extract-cbd-shape`